src/migrations/m230214_081325_add_reply_to_schema_property_2_app_dmstr_contact_template_table.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use yii\db\Migration;
+
+class m230214_081325_add_reply_to_schema_property_2_app_dmstr_contact_template_table extends Migration
+{
+    public function safeUp()
+    {
+
+        $this->addColumn('app_dmstr_contact_template','reply_to_schema_property', $this->string()->null()->defaultValue(null)->after('return_path'));
```

Comment:
Please use {{%dmstr_contact_template}} instead of specifying the prefix directly. This could lead to potential side-effects

Reply:
see above.